spec/game_spec.rb

```diff
@@ -52,4 +47,54 @@
       expect(game.current_player.mark).to eq('x')
     end
   end
+
+  context 'Checks if user can continue playing' do
+    display = Display.new
+    player1 = Player.new('x')
+    player2 = Player.new('o')
+
+    it 'when less than 9 moves have been played, can_continue_playing returns true' do
+      allow($stdin).to receive(:gets).and_return('9')
+      board = Board.new([1, 'x', 'x', 'o', 5, 6, 'o', 8, 9])
+      game = Game.new(board, display, player1, player2)
+      game.play_move
+      expect(game.can_continue_playing?).to eq(true)
+    end
+
+    it 'when 9 moves have been played, can_continue_playing returns false' do
+      allow($stdin).to receive(:gets).and_return('9')
+      board = Board.new(['o', 'x', 'x', 'o', 'x', 'o', 'x', 'o', 9])
+      game = Game.new(board, display, player1, player2)
+      game.play_move
+      expect(game.can_continue_playing?).to eq(false)
+    end
+  end
```

Comment:
During my apprenticeship I was told to visually split the tests into three parts, let's call them: preparation for the test, a call that triggest the part of logic we're testing and expectation. In your case that would be:

```
    it 'when 9 moves have been played, can_continue_playing returns false' do
      allow($stdin).to receive(:gets).and_return('9')
      board = Board.new(['o', 'x', 'x', 'o', 'x', 'o', 'x', 'o', 9])
      game = Game.new(board, display, player1, player2)

      game.play_move

      expect(game.can_continue_playing?).to eq(false)
    end
```

It makes tests faster to read.

Reply:
Thank you. I'll format the tests as shown in your example